flask/requirements-dev.txt

```diff
@@ -2,3 +2,5 @@ black==19.10b0
 flake8==3.8.1
 pre-commit==2.4.0
 pytest==5.4.3
+requests
+pytest-mock
```

Comment:
I couldn't find a module that imports pytest-mock, is it still a requirement?

Reply:
its not imported but required by pytest for the mocker fixture. I don't think I've actually used it and I'll remove it but its a good thing to look into.